src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -1,66 +1,116 @@
+        valoriHomePage.clickMobileInDropDown();
         assertTrue("Expecting the full stack header to be visible", mobileChapterPage.isMobilePageVisibile());
     }
 
-    @Then( "^I can read articles about the Mobile Chapter$" )
-    public void iCanReadAboutMobileChapter(){
+    @Then("^I can read articles about the Mobile Chapter$")
+    public void iCanReadAboutMobileChapter() {
         MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
         assertTrue(mobileChapterPage.areArticlesVisible());
     }
 
-    @And( "^I contact Valori$" )
-    public void iContactValori()throws Throwable{
-        MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
+    @And("^I contact Valori$")
+    public void iContactValori() {
+        ValoriTemplate page = new ValoriTemplate(driver);
         ContactUsPage contactUsPage = new ContactUsPage(driver);
-        mobileChapterPage.clickContactUsButton();
+        page.clickContactUsButton();
 
         //switch to new tab
-        ArrayList<String> tabs = new ArrayList<String>(driver.getWindowHandles());
+        ArrayList<String> tabs = new ArrayList<>(driver.getWindowHandles());
         driver.switchTo().window(tabs.get(1));
         assertTrue(contactUsPage.isTitleVisible());
     }
 
-    @Then( "^the contact us form is available$" )
-    public void theContactUsFormIsAvailable() throws Throwable {
-        //TODO IMPLEMENT THIS
-        //Verify the contact form fields are available.
-        throw new PendingException();
+    @Then("^the contact us form is available$")
+    public void theContactUsFormIsAvailable() {
+        ContactUsPage contactUsPage = new ContactUsPage(driver);
+        assertTrue(contactUsPage.isSendButtonEnabled());
     }
-}
 
+    @And("^I navigate to the performance testing page$")
+    public void iNavigateToThePerformanceTestingPage() {
+        ValoriHomePage valoriHomePage = new ValoriHomePage(driver);
+        PerformanceTesten performanceTestenPage = new PerformanceTesten(driver);
+
+        valoriHomePage.clickExpertisesDropdown();
+        valoriHomePage.clickPerformanceInDropDown();
+
+        assertTrue("Expecting the full stack header to be visible", performanceTestenPage.isPerformancePageSloganVisibile());
+    }
 
+    @And("^I fill in the form$")
+    public void iFillInTheForm() {
+        ContactUsPage contactUsPage = new ContactUsPage(driver);
+        contactUsPage.fillForm(persona);
+    }
+
+    @Then("^the contact send button is clickable$")
+    public void theContactSendButtonIsClickable() {
+        // this is a copy of theContactUsFormIsAvailable() due to that we don't want any side-effects on production
+        // (that would be mail to our colleagues)
+        ContactUsPage contactUsPage = new ContactUsPage(driver);
+        assertTrue(contactUsPage.isSendButtonEnabled());
+    }
+
+    @And("^I get random persona from random Api$")
+    public void apiCall() throws Exception {
```

Comment:
apiCall is een beetje nikszeggend voor een methodenaam. Wat doet de methode? Wat returned deze methode ten opzichte van andere potentiële apiCalls.

Reply:
True. Gelukkig is dit een hele makkelijke om aan te passen